test_Sender.cpp

```diff
@@ -0,0 +1,28 @@
+#define CATCH_CONFIG_MAIN 
+
+#include "test/catch.hpp"
+#include "Sender/Sender.h"
+
```

Comment:
There is no test for the output format. The crucial functionality of the Sender is to output on the console, so the receiver can read and parse it.
Writing a test to read and parse has advantages:
1. If a maintainer alters the format in future, it will alert them quickly, instead of waiting till the integration with the receiver
2. It gives you a perspective of 'how difficult it will be' for someone to parse your output - tests give you a perspective to see it from a consumer's point of view.